src/Mobs/Path.cpp

```diff
@@ -67,22 +64,8 @@ cPath::cPath(
 
 	m_NearestPointToTarget = GetCell(m_Source);
 	m_Status = ePathFinderStatus::CALCULATING;
-	m_StepsLeft = a_MaxSteps;
-
-	ProcessCell(GetCell(a_StartingPoint), nullptr, 0);
-	m_Chunk = nullptr;
-}
-
 
-
-
-
-cPath::~cPath()
-{
-	if (m_Status == ePathFinderStatus::CALCULATING)
-	{
-		FinishCalculation();
-	}
+	ProcessCell(GetCell(m_Source), nullptr, 0);
```

Comment:
From the Pathfinder's perspective, the bounding box is at one of the bottom corners (I don't remember which one right now), the game's bounding box is centered, so I adjust the point before feeding it to the Pathfinder.